monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -759,10 +759,10 @@ abstract class Observable[+A] extends Serializable { self =>
     * @param sizeOf is the function to compute the weight of each
     *        element in the buffer
     */
-  final def bufferTimedWithPressure(
+  final def bufferTimedWithPressure[AA >: A](
```

Comment:
Why dotty requires `AA >: A` ?

Reply:
Because `A` is in contravariant position here, and Scala 2 didn't recognize that.